apps/web/src/data/ecosystem.json

```diff
@@ -1,12 +1,20 @@
 [
   {
+  "name": "The Innovation Game",
+  "description": "The home of algorithmic innovation. Accelerate tech and science through innovative algorithm optimisation for real-world solutions.",
+  "url": "https://www.tig.foundation/home",
+  "imageUrl": "/images/partners/the-innovation-game.png",
+  "category": "infra",
+  "subcategory": "ai"
+},
     "name": "Moxie Protocol",
     "description": "Moxie is a SocialFi protocol on the Base blockchain that allows users to create and interact with Fan Tokens.",
     "url": "https://moxie.xyz",
     "imageUrl": "/images/partners/moxie.png",
     "category": "consumer",
     "subcategory": "social"
   },
+
```

Comment:
@gabonaut Hello, there shouldn't be a line break here, please fix. 🙏

Reply:
Apologies - resolved!